.php_cs

```diff
@@ -1,11 +1,13 @@
 <?php
```

Comment:
strict types?

Reply:
ok, so I should add this file to the fixer then... :)

Reply:
Yes :-)